src/main/java/com/charan/parkingsystem/strategy/CarStrategy.java

```diff
@@ -0,0 +1,25 @@
+package com.charan.parkingsystem.strategy;
+
+import com.charan.parkingsystem.parking.ParkingFloor;
+import com.charan.parkingsystem.parking.ParkingSlot;
+import com.charan.parkingsystem.parking.Ticket;
+import com.charan.parkingsystem.vehicle.Vehicle;
+
+import java.util.List;
+
+public class CarStrategy implements ParkStrategy {
+
+    public Ticket park(Vehicle vehicle, List<ParkingFloor> floorList){
+        Ticket ticket = null;
+        for (ParkingFloor floor : floorList) {
+            for (ParkingSlot slot: floor.getFloorSlots()) {
```

Comment:
Try to use functional programming features provided by Java8 to avoid nested loops